modules/aws/aurora/variables.tf

```diff
@@ -0,0 +1,133 @@
+variable "default_tags" {
+  description = "Default Tags"
+  type        = map(string)
+}
+
+variable "aws_region" {
+  description = "aws region"
+  type        = string
+}
+
+variable "storage_encrypted" {
+  description = "Specifies whether the DB cluster storage is encrypted. The default is `true`"
+  type        = bool
+  default     = true
+}
+
+variable "cluster_engine" {
+  description = "The name of the database engine to be used for this DB cluster. Valid Values: `aurora-mysql`, `aurora-postgresql`"
+  type        = string
+}
+
+variable "cluster_name" {
+  description = "The name of the database cluster"
+  type        = string
+}
+
+variable "engine_version" {
+  description = "The database engine version. Updating this argument results in an outage"
+  type        = string
+}
+
+variable "database_name" {
+  description = "Name for an automatically created database on cluster creation"
+  type        = string
+}
+
+variable "master_username" {
+  description = "Username for the master DB user"
+  type        = string
+}
+
+variable "master_password" {
+  description = "Password for the master DB user. Note - when specifying a value here, 'create_random_password' should be set to `false`"
+  type        = string
+  sensitive   = true
+}
+
+variable "backup_retention_period" {
+  description = "The days to retain backups for. Default `7`"
+  type        = number
+  default     = 7
+}
+
+# aws_db_subnet_group
+variable "create_db_subnet_group" {
+  description = "Determines whether to create the database subnet group or use existing"
+  type        = bool
+  default     = true
```

Comment:
should be false and we should reuse the subnets in the vpc created for the environment

Reply:
We are using exisiting subnetes created by VPC Module, Here in Aurora/RDS we reuse them and created a subnet group. This is what we pass it to RDS. RDS takes subnetgroup as input not directly subnetids